esper/query/management/commands/embed_faces.py

```diff
@@ -0,0 +1,89 @@
+from django.core.management.base import BaseCommand
```

Comment:
What's the difference between `face_embed.py`, `embed_faces.py`, and `face_cluster.py`?